packages/shared/routes/content.js

```diff
@@ -9,29 +9,29 @@ const podcast = require('../templates/content/podcast');
 const webinar = require('../templates/content/webinar');
 const document = require('../templates/content/document');
 
-module.exports = (app) => {
+module.exports = (app, siteConfig) => {
```

Comment:
`app.locals.site` is the site config object, so you should be able to do 
```js
const { site } = app.locals;
const alias = site.get('leaders.alias');
```
and skip passing the extra argument through. If you would, test and make this change for the search route as well to keep the function signatures consistent.

As with the fragments above, rename the files and variables to indicate their status as generators/functions/factories.
```suggestion
module.exports = (app, siteConfig) => {
  const { site } = app.locals;
  const companyQueryFragment = companyFragmentFn(site.get('leaders.alias'));
  const contentQueryFragment = contentFragmentFn(site.get('leaders.alias'));
  
```